### PR TITLE
Add README.md with screenshots

### PR DESCRIPTION
**Pull request by loociano**

### CODE REVIEW

File: mbv2/mbv2print/Makefile [hunk 1]
```diff
@@ -15,7 +15,7 @@ include $(DEVKITARM)/gba_rules
 # SOURCES is a list of directories containing source code
 # INCLUDES is a list of directories containing extra header files
 #---------------------------------------------------------------------------------
-TARGET		:=	sprite
+TARGET		:=	mbv2print
```
Comment (loociano) [inline mbv2/mbv2print/Makefile:18]: This name seems more appropriate. The example prints a string over mbv2, not a sprite.

**Comment by loociano** [comment]
Preview: https://github.com/devkitPro/gba-examples/blob/0eec18d120973cbc7024764692cf3e2561bc9623/README.md

**Comment by loociano** [comment]
Hey @WinterMute, let me know what you think!

**Comment by HorstBaerbel** [comment]
This repo might also need a license too...

**Comment by WinterMute** [comment]
@loociano Sorry to take so long replying to this.

It's hard to know what to do with this - some of the examples here are very outdated and depend on devices that are no longer available. libgba itself was never really finished - I moved on to DS then later 3DS & now we're working with switch homebrew.  The bg scroller in particular is cringeworthy, mbv2 is no longer available and the lack of parallel ports on modern hardware put paid to the ghetto xboo cables we used to use .

It would be nice to revisit some of the old code & do a bit of a refresh, especially now we have the luxury of pacman & the ability to package things without having to rewrite an installer. libtonc is/was much more extensive than libgba as well as having much better documentation (i.e. being documented at all :p). Cearn is happy to have libtonc distributed

The situation requires some thought

**Comment by HorstBaerbel** [comment]
Just my 50ct: For demoscene and retro games development this is still relevant, though people are now using flashcarts and emulators. For me libga should provide the minimum amount of convenience: easier hardware register access and register bit definitions, some struct typedefs, interrupt handling and not much more. Very low level stuff. If you want a full-blown sprite engine etc., go with TONC.
Also the MaxMOD example is still relevant too.

**Comment by WinterMute** [comment]
1. It's possible to use maxmod with libtonc.
2. libtonc doesn't contain a full blown sprite engine, it's really not that much higher level than libgba and has better documentation.

**Comment by WinterMute** [comment]
Actually, I'm going to merge this for now & worry about cleaning up old projects later.

Thanks for the PR, much appreciated.

**Comment by loociano** [comment]
No problem at all @WinterMute, thanks for reviewing!

**Pull request merged**